docs/dev-machine-setup.md

```diff
@@ -15,7 +15,56 @@ outlined in the [Getting Started](getting-started.md) instructions).
 
 ### Local Virtual Machine
 
-`[TODO]`
+Firecracker can also be installed in a Proxmox VM which uses KVM as Hypervisor. The nested option must be enabled on the host where Proxmox is running. In the following example nested virtualization has not been enabled yet ("N" No) so to check this in your Proxmox host excecute the following command:
```

Comment:
I'd add a new heading for Proxmox.
```
#### Intel Proxmox
```

Reply:
I'd also like to keep the markdown style as consistent as possible even though it's not strictly enforced yet. Please limit the lines to 80 chars (maybe use a linter like [this one](https://github.com/markdownlint/markdownlint) to check the style locally before pushing).